packages/subgraph/package.json

```diff
@@ -20,14 +20,15 @@
     "clean": "rimraf deploy-output.txt subgraph.yaml ./build ./imported ./generated ./tests/.bin tests/.latest.json && yarn postinstall"
   },
   "devDependencies": {
-    "osx-ethersV120": "npm:@aragon/osx-ethers@1.2.1",
     "@graphprotocol/graph-cli": "^0.51.0",
     "@graphprotocol/graph-ts": "^0.31.0",
     "cross-env": "^7.0.3",
     "matchstick-as": "^0.5.2",
     "mustache": "^4.2.0",
+    "osx-ethersV120": "npm:@aragon/osx-ethers@1.2.1",
```

Comment:
```suggestion
```
I think you included this in the beginning and then realized that you find the repo in the `plugin-info.json`. At least I don't see it used anywhere.

Reply:
It is used here `packages/subgraph/scripts/postInstall.ts`

Reply:
then we need to update to 1.3.0